spec/models/ability_spec.rb

```diff
@@ -403,4 +403,18 @@
       expect(ability).to be_able_to(:join, helpdesk_team)
     end
   end
+
+  context 'Crud Conferences' do
```

Comment:
Yay, looks already really cool 👍 

Do these specs also work if you use `build_stubbed` instead of `create` (would speed them up 😉 )
Plus: it would be convenient to also add a negative example, where a user cannot crud conferences.